pkg/convert/pot.go

```diff
@@ -73,33 +49,21 @@ func FromPot(b []byte) (model.Messages, error) {
 	}
 
 	messages := make([]model.Message, 0, len(po.Messages))
-	// model.Messages does not support plurals atm. But we plan to impl it under:
-	// https://github.com/orgs/translate-agent/projects/1?pane=issue&itemId=21251425
-	for _, node := range po.Messages {
-		if strings.HasSuffix(node.MsgId, "\\n") {
-			node.MsgId = strings.ReplaceAll(node.MsgId, "\\n", "\n")
-		}
-
-		if node.MsgIdPlural != "" {
-			continue
-		}
-
-		fuzzy := strings.Contains(node.Flag, "fuzzy")
-
-		message := model.Message{
-			ID:          node.MsgId,
-			Description: strings.Join(node.ExtractedComment, "\n"),
-			Fuzzy:       fuzzy,
-		}
-
-		if len(node.MsgStr) > 0 {
-			if strings.HasSuffix(node.MsgStr[0], "\\n") {
-				node.MsgStr[0] = strings.ReplaceAll(node.MsgStr[0], "\\n", "\n")
-			}
 
+	for _, node := range po.Messages {
+		var message model.Message
+		if po.Header.PluralForms.NPlurals > 1 { // Handle messages with plural forms
+			message.ID = node.MsgId
+			message.PluralID = node.MsgIdPlural
+			message.Message = convertToFormatMessage(node.MsgStr)
+		} else {
+			message.ID = node.MsgId
 			message.Message = node.MsgStr[0]
```

Comment:
You can combine all same fields for struct,
```suggestion
		message := model.Message{ID:node.MsgId}
		if po.Header.PluralForms.NPlurals > 1 { // Handle messages with plural forms
			message.PluralID = node.MsgIdPlural
			message.Message = convertToFormatMessage(node.MsgStr)
		} else {
			message.Message = node.MsgStr[0]
```